about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Browse Caitlin's abstract art">
+    <meta name="keywords" content="zentangles, abstract, art, drawing">
+
+    <title>Caitlin's Art</title>
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="container">
+      <div class="grey-background">
+        <a name="top"></a>
+        <a href="index.html"><img src="images/personal-logo-transparent.png" class="logo"></a>
+        <div class="navigation">
+          <ul>
+            <li><a href="about.html">about</a></li>
+            <li><a href="art.html">art</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="about-me">
+        <img src="images/self2.png" class="about-image greyscale">
+        <h1>About the Artist</h1>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus vulputate elit auctor, sodales ipsum sed, mollis nisl. Cras ultrices quam et orci lacinia, eu aliquet diam condimentum. In gravida eu leo at molestie. Aliquam sit amet urna augue. Pellentesque condimentum vestibulum lacus, sit amet varius sem finibus a. In hac habitasse platea dictumst. In condimentum dolor ipsum, nec varius lectus tristique elementum. Duis scelerisque, odio quis pharetra sagittis, massa enim egestas tortor, sagittis pellentesque dui est sit amet nisl.
```

Comment:
Start filling this content in! I know it seems like something you can do at the last minute but sometimes our content informs our design.